src/Catty/Settings.bundle/Root.plist

```diff
@@ -30,7 +30,7 @@
 		</dict>
 		<dict>
 			<key>DefaultValue</key>
-			<string>0.6.10</string>
+			<string>0.6.11</string>
```

Comment:
did you pull from develop before submitting the PR? I am pretty sure, that on develop there is already this change..